.gitignore

```diff
@@ -2,6 +2,4 @@
 *.rej
 *.native
 _build
-setup.ml
-setup.data
-setup.log
+.merlin
```

Comment:
Do you need to add `*.install` here? It may be not necessary because I cannot see install targets but better to double check. Does `jbuilder build` create a `xen-api-sdk.install` file?